pkg/zson/docs/spec.md

```diff
@@ -419,34 +414,34 @@ The formats for each type is as follows:
 
 Type | Format
 ---- | ------
-bool | a single characeter "t" or "f"
-string | a utf-8 string that may optionally include escape sequences
-bytes | a sequence of raw bytes encoded as base64
-int | decimal string representation of any signed, 64-bit integer
-count | decimal string of any unsigned, 64-bit integer
-double | a decimal representation of a 64-bit IEEE floating point literal as defined in javascript
-time | unsigned dotted decimal notation of seconds (32-bit second, 32-bit nanosecond)
-interval | signed dotted decimal notation of seconds (32-bit second, 32-bit nanosecond)
-port | an integer string in [0,65535] with an option suffix of "/udp" or "/tcp"
-addr | a string representing a numeric in IPv4 address form or IPv6 form
-subnet | a string in CIDR notation representing an IP address and prefix length as defined in RFC 4632 and RFC 4291.
-enum | a string representing an enum value defined outside the scope of zson
-
-*Note: a string can embed binary data using escapes.  It's up to the receiver to determine
+`bool` | a single characeter `T` or `F`
```

Comment:
Once again, assuming "legacy" is synonymous with "the way Zeek behaves today", these should be uppercase. Attempting to make Zeek scripting use lowercase `t` or `f` in a `bool` type today causes it to choke, so we'd never see these output in lowercase in current Zeek ASCII TSV logs.

Reply:
Consensus is T/F are fine for now, and we could always relax this later to support more formats.